modules/securitygroups.py

```diff
@@ -15,14 +15,18 @@ def scan():
         used_groups.extend(attached)
 
     all_sg = get_all_sg()
-
+    elb_sg = get_elb_sg()
     print("Found {} security groups".format(len(all_sg)))
 
     not_used = []
     for group in all_sg:
         id = group["GroupId"]
         if id not in used_groups:
-            not_used.append(group)
+            if len(elb_sg) > 0:
+                if id not in elb_sg:
```

Comment:
We already have a `used_groups` array with all SGs that are in use by anything. Just append the list of your SGs from the ELBs to this list and then you don't need to modify this part of the code